src/man.rs

```diff
@@ -89,6 +131,18 @@ impl Manual {
     self
   }
 
+  pub fn exit_status(mut self, exit_status: ExitStatus) -> Self {
+    match exit_status {
+      ExitStatus { code: None, .. } => {
+        self.exit_statuses.set_to_default();
```

Comment:
Ah, so this line is what I was missing! I didn't get when you would ever set the exit status list to the default list – I assumed it was the set you were starting with and then when you add a custom one it gets overwritten.

This behavior here is surprising to me: When you call this public function, that has no docs, with a "special" value (`code = None`), it does an unusual thing. What's also weird: To get this special input you'd have to call `default()`, but from what I can tell this "special" mode should not be the "default".